packages/create-discord-bot/package.json

```diff
@@ -0,0 +1,69 @@
+{
+	"name": "create-discord-bot",
+	"version": "0.1.0",
+	"description": "A simple way to create a startup Discord bot.",
+	"keywords": [
+		"discord",
+		"discord-api",
+		"discord.js",
+		"create-bot",
+		"init-bot",
+		"init-discord",
+		"boilerplate"
+	],
+	"homepage": "https://discord.js.org",
+	"bugs": {
+		"url": "https://github.com/discordjs/discord.js/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/discordjs/discord.js.git"
+	},
+	"license": "Apache-2.0",
+	"contributors": [
+		"Crawl <icrawltogo@gmail.com>",
+		"SpaceEEC <spaceeec@yahoo.com>",
+		"Vlad Frangu <kingdgrizzle@gmail.com>",
+		"Aura Román <kyradiscord@gmail.com>",
+		"Jiralite <jiralite@live.co.uk>"
+	],
+	"bin": "./dist/index.mjs",
+	"directories": {
+		"lib": "src"
+	},
+	"files": [
+		"dist",
+		"template"
+	],
+	"scripts": {
+		"test": "tsup",
```

Comment:
is this inteded?

Reply:
Should be resolved with c4fcf6b92c900218b1480f20deeca2e64c14afb0.